coreneuron/sim/scopmath/crout_thread.hpp

```diff
@@ -0,0 +1,235 @@
+/******************************************************************************
+ *
+ * File: crout.c
+ *
+ * Copyright (c) 1987, 1988, 1989, 1990
+ *   Duke University
+ *
+ ******************************************************************************/
+#pragma once
+#include "coreneuron/sim/scopmath/errcodes.h"
+#include "coreneuron/sim/scopmath/newton_struct.h"
+
+// #include <cmath>
```

Comment:
`nitpick`: remove?

By the way, were there any logical changes in this file? (asking as Github doesn't show this as mere renaming)

Reply:
Removed that #include, see https://github.com/BlueBrain/CoreNeuron/pull/809#discussion_r865613379 for a description of the changes in these files.